README.md

```diff
@@ -649,4 +649,4 @@ It covers different usages:
 2. from a request scoped service
 3. from a REST controller endpoint (using `@RestController)
 
```

Comment:
Missing information about the new module in the README.md.

Reply:
Putted in some information